angular/src/app/shared/data-table/data-table.component.html

```diff
@@ -13,3 +14,9 @@
                  [multiSortKey]="defaultTableConfig.multiSortKey"
                  [navigateToNextCell]="navigateToNextCell">
 </ag-grid-angular>
+<div style="align-self: flex-end;">
```

Comment:
Please see "semantic CSS" in https://docs.google.com/document/d/1fQz5u6oM56qZ2hinDvPklexG6qvnu-jcymdhU5NblVQ/edit